config/CUSTOM-RULES.md

```diff
@@ -151,14 +177,61 @@ In-battle rules
 
 `Inverse Mod` - inverse type effectiveness (like in Gen 6 Inverse Battles)
 
+`Gen 8 Camomons` - Pokémon will change their typing to match their first two moveslots
+
+`Gen 7 Tier Shift` - Pokémon will have higher base stats the lower their Gen 7 Smogon tier is
+
+`Dynamax Clause` - prevent Pokémon from Dynamaxing during battle. Cannot be used to allow Dynamaxing in old gens
+
+`Endless Battle Clause` - prevent battles from proceeding endlessly
+
+`HP Percentage Mod` - Show the opposing Pokémon's HP rounded to the nearest percent, as opposed to a range of percentages based upon the health bar's size in-game
+
+`Exact HP Mod` - Show the opposing Pokémon's HP rounded to the nearest tenth of a percent
+
+`Cancel Mod` - allow players to cancel their moves
+
+`Switch Priority Clause Mod` - make the fastest Pokémon switch first when more than one Pokémon switches out at once, unlike in Emerald link battles, where Player 1's Pokémon would switch first.
+
+
+Standard Rulesets
+-----------------
+
```

Comment:
```suggestion

Most formats already come with a standard ruleset. To change which one you use in a tournament, you would have to do something like `!Standard, Standard Ubers`.

```

Reply:
the issues with this are being fixed soon so sure